plugins/obs-qsv11/obs-qsv11.c

```diff
@@ -106,13 +108,35 @@ static unsigned short g_verMinor;
 static int64_t g_pts2dtsShift;
 static int64_t g_prevDts;
 static bool g_bFirst;
+static qsv_adapter_info g_new_adapter_info;
 
 static const char *obs_qsv_getname(void *type_data)
 {
 	UNUSED_PARAMETER(type_data);
 	return "QuickSync H.264";
 }
 
+static const char *obs_qsv_getname_av1(void *type_data)
+{
+	UNUSED_PARAMETER(type_data);
+	return "QuickSync AV1";
+}
+
+bool av1_supported(mfxIMPL impl)
+{
+	return qsv_av1_supported(impl);
+}
+
+bool check_av1_adapter()
+{
+	//av1_soft is non texture based encode path
```

Comment:
Leave a space between the `//` and the actual comment.
Use a hyphen for "non-texture".